Code/Mantid/MantidQt/CustomInterfaces/src/TomoReconstruction/TomoReconstruction.cpp

```diff
@@ -427,17 +438,17 @@ void TomoReconstruction::setupComputeResource() {
     const Mantid::Kernel::FacilityInfo &fac =
         Mantid::Kernel::ConfigService::Instance().getFacility();
     if (fac.name() != m_facility) {
-      userError("Facility not supported",
-                "This interface is designed "
-                "to be used at " +
-                    m_facility +
-                    ". You will probably not be "
-                    "able to use it in a useful way because your facility "
-                    "is " +
-                    fac.name() +
-                    ". If you have set that facility "
-                    "facility by mistake in your settings, please update it.");
-      return;
+      g_log.error()
+          << "Facility not supported. This interface is designed "
+             "to be used at " +
+                 m_facility +
+                 ". You will probably not be able to use it in a useful way "
+                 "because your facility is " +
+                 fac.name() +
+                 ". If you have set that facility by mistake in your settings, "
+                 "please update it." << std::endl;
+      throw std::runtime_error(
+          "Failed to initialize because the facility is not " + fac.name());
```

Comment:
Should  ```fac.name()``` not be ```m_facilty``` (or ```facility is not``` be ```facility is```)?

Currently I get: ```Error description: Failed to initialize because the facility is not SNS```